casadm/statistics_model.c

```diff
@@ -632,20 +632,26 @@ static int cache_stats(int ctrl_fd, const struct kcas_cache_info *cache_info,
 		      bool by_id_path)
 {
 	struct kcas_get_stats cache_stats = {};
+	bool standby;
+
 	cache_stats.cache_id = cache_id;
 	cache_stats.core_id = OCF_CORE_ID_INVALID;
 	cache_stats.part_id = OCF_IO_CLASS_INVALID;
 
-	if (ioctl(ctrl_fd, KCAS_IOCTL_GET_STATS, &cache_stats) < 0)
+	standby = !!(cache_info->info.state & (1 << ocf_cache_state_standby));
```

Comment:
Perhaps a function/macro similar to ocf_cache_is_standby()  would be better here? We might then refactor many other places in casadm to use such macro :)

Reply:
I would happily do such refactor, but I don't have time for this at the moment. I suggest to leave it as is for now.

Reply:
... as this would require changing multiple lines of code